common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -863,8 +884,8 @@ public void onComplete(String streamId) throws IOException {
         if (isStaleOrTooLate(appShuffleInfo.shuffles.get(partitionInfo.shuffleId),
             partitionInfo.shuffleMergeId, partitionInfo.reduceId)) {
           deferredBufs = null;
-          throw new RuntimeException(String.format("Block %s is %s", streamId,
-            ErrorHandler.BlockPushErrorHandler.TOO_LATE_OR_STALE_BLOCK_PUSH_MESSAGE_SUFFIX));
+          throw new RuntimeException("Block " + streamId + " " +
```

Comment:
There are still other instances in this class where we are formatting the message of the RuntimeException with `String.format` but those haven't been modified. Is it because they are quite infrequent? Still would be nice to have this consistent in this class.

Reply:
Marking as resolved since we no longer build a string here in the latest version.

Reply:
@Victsm there is still another instance in `RemoteBlockPushResolver` that throws `StaleBlockPushException` 
```
            throw new StaleBlockPushException(String.format("Rejecting shuffle blocks push request"
              + " for shuffle %s with shuffleMergeId %s for application %s_%s as a higher"
              + " shuffleMergeId %s request is already seen", shuffleId, shuffleMergeId,
              appShuffleInfo.appId, appShuffleInfo.attemptId, latestShuffleMergeId));
 ```